packages/app/src/components/button/filled/index.tsx

```diff
@@ -89,4 +89,62 @@ const FilledButton = function <T = null>({
     </button>
   );
 };
+
+const Renewal = function <T = null>({
```

Comment:
The component named Renewal will be offered this way until all design updates are complete.

When all changes are complete, replace the original component with the Renewal component.